app/views/submissions/browse.html.erb

```diff
@@ -8,7 +8,9 @@
           show_sections: <%= Section.exists? %>,
           is_admin: <%= @current_user.admin? %>,
```

Comment:
please double check this but I don't think the `is_admin` and `is_ta` props are being used anymore so they should be removed.

Reply:
We are using is_admin props but not is_ta. I removed is_ta props.